index.js

```diff
@@ -2,12 +2,23 @@
 const {ContextualIdentityService} = require('resource://gre/modules/ContextualIdentityService.jsm');
 const { Cc, Ci, Cu, Cr } = require('chrome');
 
+Cu.import("resource://gre/modules/Services.jsm");
+
 const tabs = require('sdk/tabs');
 const webExtension = require('sdk/webextension');
 const { viewFor } = require("sdk/view/core");
 var windowUtils = require('sdk/window/utils');
 var tabsUtils = require('sdk/tabs/utils');
 
+const { browserWindows: windows } = require("sdk/windows");
+const { attachTo } = require("sdk/content/mod");
+const { Style } = require("sdk/stylesheet/style");
+let self = require("sdk/self");
+let style = Style({ uri: self.data.url("chrome.css") });
+
+for (let w of windows)
```

Comment:
1. This should go into init()
2. use {}
3. add a comment about styling windows.